### PR TITLE
General Improvements

### PR DESCRIPTION
**Pull request by athoscouto**
A series of small improvements that would be too little to fit in a single PR.

Please test it with https://github.com/vtex-apps/admin-messages/pull/7, since the later depends on this one.

### CODE REVIEW

File: react/components/AdminMessages.tsx [hunk 1]
```diff
@@ -76,7 +79,7 @@ const AdminMessages: FC<Props> = ({ addProductTranslations, email, intl }) => {
             messages={messages!}
             setMessages={setMessages}
           />
-        )}
+        ) : null}
```
Comment (felippenardi) [inline react/components/AdminMessages.tsx]: @athoscouto Muda os ternários para `&&`:

```js
{ step && (
	<ProductMessagesProgress />
)}
```

File: react/components/ProductMessagesExport.tsx [hunk 2]
```diff
@@ -1,9 +1,61 @@
...
 import { StepCounterControl } from '../typings/typings'
 import StepCounter from './StepCounter'
 
+
+const {
+  inst1Message,
+  inst2Message,
+  inst3Message,
+  inst4Message,
+  inst5Message,
```
Comment (felippenardi) [inline react/components/ProductMessagesExport.tsx]: @athoscouto Ficou bem difícil eu entender que vc queria dizer "instruction" aqui, vc pode renomear pra ficar mais verboso?

File: react/components/ProductMessagesProgress.tsx [hunk 3]
```diff
@@ -36,20 +73,20 @@ function getUploadProgressMessage(
...
     return intl.formatMessage(
-      { id: 'admin/messages.messages-progress.translate-failed' },
+      translateFailedMessage,
       { failed }
     )
   }
 
   return intl.formatMessage(
-    { id: 'admin/messages.messages-progress.translate-success' },
+    translateSuccessMessage,
```
Comment (felippenardi) [inline react/components/ProductMessagesProgress.tsx]: non-blocking: better name is `translationFailed`

File: react/components/ProductMessagesProgress.tsx [hunk 4]
```diff
@@ -147,15 +184,15 @@ const ProductMessagesImport: FC<Props> = ({
         <p>{getCurrentProgressPercentage(progress, total)}</p>
         {progress < total ? (
           <Button variation="secondary" onClick={cancel}>
-            {intl.formatMessage({ id: 'admin/messages.cancel' })}
+            {intl.formatMessage(cancelMessage)}
```
Comment (felippenardi) [inline react/components/ProductMessagesProgress.tsx]: non-blocking: all intl are messages, so you don't need to repeat "Message" in the translation key. `cancel` alone would suffice

File: react/graphql/AddProductTranslations.graphql [hunk 5]
```diff
@@ -1,5 +1,6 @@
 mutation AddProductTranslations($translations: [MessagesFromProvider]!, $language: String!) {
   addProductTranslations(translations: $translations, language: $language) {
+    # Returns provider of failed translations
```
Comment (felippenardi) [inline react/graphql/AddProductTranslations.graphql]: Let's talk about this structure. We should not be using `data` to communicate failures

**Comment by athoscouto** [comment]
Hey @felippenardi, addressed most of your requests on https://github.com/vtex-apps/admin-messages/pull/9/commits/7515500939a7a171ba87a63cdbf242bff906e440 and onhttps://github.com/vtex-apps/admin-messages/pull/9/commits/36a6cbffc947bf677ede7f8a5e96e8d5ccb60047.
We should talk about handling errors on batch operations, but for now I'll leave it like this.

**Pull request merged**